<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chess</class>
 <widget class="QMainWindow" name="Chess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="windows">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>633</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="startPage">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>325</y>
        <width>350</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>450</y>
        <width>350</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>800</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold SemiConden</family>
        <pointsize>70</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Câ™œESS  PUZZLES</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="mainPageRows">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>25</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>25</number>
       </property>
       <item row="3" column="0">
        <widget class="QPushButton" name="lesson4">
         <property name="text">
          <string>Overloading</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="puzzle4">
         <property name="text">
          <string>Distract The Queen</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="lesson5">
         <property name="text">
          <string>Controling The King</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="puzzle5">
         <property name="text">
          <string>Force The King</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="puzzle1">
         <property name="text">
          <string>Find The Fork</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="lesson1">
         <property name="text">
          <string>Fork</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="lesson3">
         <property name="text">
          <string>Smother Mate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="puzzle2">
         <property name="text">
          <string>Pin The Queen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="lesson2">
         <property name="text">
          <string>Pinning</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="puzzle3">
         <property name="text">
          <string>Smother Mate The King</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="lesson6">
         <property name="text">
          <string>Combining Tactics</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="puzzle6">
         <property name="text">
          <string>Find The Fork And Pins</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="basicGameTemplate">
     <widget class="Puzzle" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>230</y>
        <width>640</width>
        <height>640</height>
       </rect>
      </property>
      <widget class="QLabel" name="feedbackLabel">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>230</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Puzzle</class>
   <extends>QWidget</extends>
   <header>puzzle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
